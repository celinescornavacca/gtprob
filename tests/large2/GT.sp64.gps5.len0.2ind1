((((((CB1,CB2),((CB3,CB4),CB0)),(DB0,((DB3,DB4),(DB1,DB2)))),(((((XA3,XA4),XA0),(XA1,XA2)),(((YA1,YA2),(YA0,(YA3,YA4))),(((ZA1,ZA2),ZA0),(ZA3,ZA4)))),(((GA0,(GA1,(GA2,(GA3,GA4)))),((((HA3,HA4),HA2),HA1),HA0)),((((WB0,(WB1,((WB3,WB4),WB2))),(XB0,((XB1,XB2),(XB3,XB4)))),((LB3,LB4),((LB1,LB2),LB0))),(((QB0,((QB2,(QB3,QB4)),QB1)),(RB0,(RB1,(RB2,(RB3,RB4))))),((((UB1,UB2),UB0),(UB3,UB4)),(((VB2,(VB3,VB4)),VB1),VB0))))))),((AA0,AA1),(AA2,(AA3,AA4)))),((((((((((IA3,IA4),IA2),IA1),IA0),((SB0,((SB3,SB4),(SB1,SB2))),((TB2,(TB3,TB4)),(TB0,TB1)))),((((JB3,JB4),JB0),(JB1,JB2)),((YB0,((YB1,YB2),(YB3,YB4))),(((ZB1,ZB2),(ZB3,ZB4)),ZB0)))),(((FA0,FA1),((FA3,FA4),FA2)),((((HB1,HB2),((HB3,HB4),HB0)),(IB0,((IB1,IB2),(IB3,IB4)))),(((((IC1,IC2),IC0),(IC3,IC4)),((JC3,JC4),(JC0,(JC1,JC2)))),(((CC3,CC4),CC2),(CC0,CC1)))))),(((((RA0,(RA1,((RA3,RA4),RA2))),((SA0,(SA3,SA4)),(SA1,SA2))),((((MB2,(MB3,MB4)),(MB0,MB1)),(NB0,(((NB3,NB4),NB2),NB1))),(((WA1,WA2),WA0),(WA3,WA4)))),((((KA0,(((KA3,KA4),KA2),KA1)),(LA0,((LA2,(LA3,LA4)),LA1))),(((((AB3,AB4),AB2),AB1),AB0),((BB1,((BB3,BB4),BB2)),BB0))),(((OA2,(OA3,OA4)),(OA0,OA1)),(((VA2,(VA3,VA4)),(VA0,VA1)),((((DC0,DC1),((DC3,DC4),DC2)),(((GC2,(GC3,GC4)),(GC0,GC1)),(((HC2,(HC3,HC4)),HC1),HC0))),(((KB1,KB2),KB0),(KB3,KB4))))))),(((((OB1,OB2),(OB0,(OB3,OB4))),(((PB2,(PB3,PB4)),PB1),PB0)),((KC0,(((KC3,KC4),KC2),KC1)),(LC0,(LC1,(LC2,(LC3,LC4)))))),((((((JA0,JA1),(JA2,(JA3,JA4))),((((EC3,EC4),EC0),(EC1,EC2)),((FC0,(FC1,FC2)),(FC3,FC4)))),(((PA0,PA1),(PA2,(PA3,PA4))),((QA1,QA2),((QA3,QA4),QA0)))),((EA0,EA1),(EA2,(EA3,EA4)))),((((EB0,((EB3,EB4),(EB1,EB2))),((AC0,(AC1,((AC3,AC4),AC2))),((BC1,BC2),((BC3,BC4),BC0)))),(((UA3,UA4),UA0),(UA1,UA2))),(TA0,((TA3,TA4),(TA1,TA2)))))))),((((DA1,DA2),(DA0,(DA3,DA4))),((MA0,((MA2,(MA3,MA4)),MA1)),(((NA3,NA4),NA2),(NA0,NA1)))),((((BA1,BA2),BA0),(BA3,BA4)),((CA0,CA1),((CA3,CA4),CA2))))),(((FB2,(FB3,FB4)),(FB0,FB1)),((GB1,(GB2,(GB3,GB4))),GB0))));
