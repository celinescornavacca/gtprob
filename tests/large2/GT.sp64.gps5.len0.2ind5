((((KA0,((KA2,(KA3,KA4)),KA1)),((LA3,LA4),(LA0,(LA1,LA2)))),((((ZB0,(((ZB3,ZB4),ZB2),ZB1)),(((AC2,(AC3,AC4)),AC1),AC0)),((((BA1,BA2),BA0),(BA3,BA4)),(CA0,(CA1,((CA3,CA4),CA2))))),((((((((KC3,KC4),KC2),KC1),KC0),(LC0,((LC1,LC2),(LC3,LC4)))),(((VA0,VA1),((VA3,VA4),VA2)),((((OB1,OB2),OB0),(OB3,OB4)),(((PB3,PB4),(PB1,PB2)),PB0)))),(((((EA3,EA4),EA2),EA1),EA0),(((XA0,(XA1,XA2)),(XA3,XA4)),((YA0,(YA3,YA4)),(YA1,YA2))))),(((((((CB3,CB4),CB0),(CB1,CB2)),(DB0,(((DB3,DB4),DB2),DB1))),(NA0,((NA3,NA4),(NA1,NA2)))),((((((PA3,PA4),((PA1,PA2),PA0)),((QA0,QA1),(QA2,(QA3,QA4)))),(((SB3,SB4),((SB1,SB2),SB0)),((TB3,TB4),(TB0,(TB1,TB2))))),(((DA3,DA4),DA0),(DA1,DA2))),(((GA1,((GA3,GA4),GA2)),GA0),((((XB0,XB1),(XB2,(XB3,XB4))),(YB0,(((YB3,YB4),YB2),YB1))),(((EB2,(EB3,EB4)),EB1),EB0))))),(((OA0,(OA1,OA2)),(OA3,OA4)),((((MB1,MB2),(MB0,(MB3,MB4))),(NB0,(NB1,(NB2,(NB3,NB4))))),((((FB1,FB2),FB0),(FB3,FB4)),((((BC3,BC4),(BC1,BC2)),BC0),(((GC0,(GC3,GC4)),(GC1,GC2)),(((HC1,HC2),(HC3,HC4)),HC0)))))))))),(((((((TA3,TA4),((TA1,TA2),TA0)),((((VB2,(VB3,VB4)),VB1),VB0),(((WB3,WB4),WB2),(WB0,WB1)))),(MA0,(MA1,((MA3,MA4),MA2)))),(((((JA1,JA2),JA0),(JA3,JA4)),((((WA1,WA2),(WA3,WA4)),WA0),((BB0,((BB3,BB4),(BB1,BB2))),((((KB3,KB4),(KB1,KB2)),KB0),((LB0,(LB3,LB4)),(LB1,LB2)))))),(((((HA3,HA4),HA2),HA1),HA0),(((QB1,((QB3,QB4),QB2)),QB0),(RB0,((RB1,RB2),(RB3,RB4))))))),(((((RA3,RA4),RA0),(RA1,RA2)),((SA1,(SA2,(SA3,SA4))),SA0)),(((GB3,GB4),(GB0,(GB1,GB2))),((HB1,((HB3,HB4),HB2)),HB0)))),(((((IB2,(IB3,IB4)),IB1),IB0),((JB3,JB4),(JB0,(JB1,JB2)))),(((AA0,AA1),((AA3,AA4),AA2)),(((((((((CC3,CC4),CC2),CC1),CC0),(((DC2,(DC3,DC4)),DC1),DC0)),((((UB3,UB4),UB2),UB1),UB0)),((IA1,IA2),(IA0,(IA3,IA4)))),((((ZA0,ZA1),(ZA2,(ZA3,ZA4))),(((AB3,AB4),AB0),(AB1,AB2))),((UA1,((UA3,UA4),UA2)),UA0))),(((FA0,(FA3,FA4)),(FA1,FA2)),(((((IC3,IC4),IC2),(IC0,IC1)),((JC0,(JC1,JC2)),(JC3,JC4))),(((EC2,(EC3,EC4)),(EC0,EC1)),((((FC3,FC4),FC2),FC1),FC0)))))))));
