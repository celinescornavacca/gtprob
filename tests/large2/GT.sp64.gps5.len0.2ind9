(((((RA0,(RA1,RA2)),(RA3,RA4)),(((SA2,(SA3,SA4)),SA1),SA0)),(((((((JA0,(JA1,((JA3,JA4),JA2))),((((((GC1,GC2),GC0),(GC3,GC4)),((HC3,HC4),((HC1,HC2),HC0))),(KB0,(KB1,((KB3,KB4),KB2)))),(((LA2,(LA3,LA4)),LA1),LA0))),(((((UA1,UA2),(UA3,UA4)),UA0),((VA0,VA1),((VA3,VA4),VA2))),((MA0,MA1),((MA3,MA4),MA2)))),((((FA0,(FA1,((FA3,FA4),FA2))),((((UB3,UB4),(UB0,(UB1,UB2))),((VB3,VB4),((VB1,VB2),VB0))),(PA0,(((PA3,PA4),PA2),PA1)))),(((EB0,(EB3,EB4)),(EB1,EB2)),((FB1,(FB2,(FB3,FB4))),FB0))),((((KA1,KA2),(KA0,(KA3,KA4))),(((((SB1,SB2),((SB3,SB4),SB0)),(((TB1,TB2),TB0),(TB3,TB4))),(((DB1,DB2),DB0),(DB3,DB4))),((ZA1,(ZA2,(ZA3,ZA4))),ZA0))),(((NB0,NB1),((NB3,NB4),NB2)),(((OB0,OB1),((OB3,OB4),OB2)),((((PB3,PB4),PB2),PB1),PB0)))))),(((CA0,(CA1,CA2)),(CA3,CA4)),((DA1,DA2),((DA3,DA4),DA0)))),(((((CB0,(CB3,CB4)),(CB1,CB2)),((((ZB0,ZB1),((ZB3,ZB4),ZB2)),((IC0,((IC1,IC2),(IC3,IC4))),((((JC3,JC4),JC2),JC1),JC0))),((((CC2,(CC3,CC4)),CC1),CC0),(((DC2,(DC3,DC4)),DC1),DC0)))),((WA0,WA1),((WA3,WA4),WA2))),(TA0,(TA1,((TA3,TA4),TA2))))),((((HB2,(HB3,HB4)),(HB0,HB1)),(IB0,((IB1,IB2),(IB3,IB4)))),((AB0,(AB1,(AB2,(AB3,AB4)))),((BB0,(BB1,BB2)),(BB3,BB4)))))),(((XA0,((XA3,XA4),(XA1,XA2))),(YA0,((YA3,YA4),(YA1,YA2)))),((((((WB0,(WB1,((WB3,WB4),WB2))),(XB0,(XB1,(XB2,(XB3,XB4))))),(((QA1,QA2),QA0),(QA3,QA4))),((HA0,HA1),(HA2,(HA3,HA4)))),((((GA3,GA4),GA2),GA1),GA0)),(((AA3,AA4),(AA0,(AA1,AA2))),((((LB0,((LB2,(LB3,LB4)),LB1)),(((MB3,MB4),MB0),(MB1,MB2))),((JB1,JB2),(JB0,(JB3,JB4)))),(((BA0,((BA1,BA2),(BA3,BA4))),(((IA0,(IA1,((IA3,IA4),IA2))),((((KC0,(KC1,KC2)),(KC3,KC4)),((LC0,(LC3,LC4)),(LC1,LC2))),(((QB0,(QB3,QB4)),(QB1,QB2)),(((RB1,RB2),(RB3,RB4)),RB0)))),(((YB0,(YB1,YB2)),(YB3,YB4)),((AC0,(AC1,(AC2,(AC3,AC4)))),(BC0,(((BC3,BC4),BC2),BC1)))))),((((EA2,(EA3,EA4)),EA1),EA0),((((NA3,NA4),(NA0,(NA1,NA2))),((OA0,OA1),(OA2,(OA3,OA4)))),((((EC3,EC4),((EC1,EC2),EC0)),(((FC3,FC4),(FC1,FC2)),FC0)),((GB1,(GB2,(GB3,GB4))),GB0))))))))));
