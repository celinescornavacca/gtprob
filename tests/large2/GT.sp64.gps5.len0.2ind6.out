-612.627630066
